src/main/java/com/javadiscord/javabot/service/AutoMod.java

```diff
@@ -34,14 +33,10 @@
     private static List<String> spamUrls;
 
     public AutoMod() {
-        try {
-            Scanner scanner = new Scanner(new File(String.valueOf(getClass().getResourceAsStream("spamsLinks.txt"))));
-            spamUrls = new ArrayList<>();
-            while (scanner.hasNext()){
-                spamUrls.add(scanner.next());
-            }
-            scanner.close();
-        } catch (FileNotFoundException e) {
+        spamUrls = new ArrayList<>();
+        try (var linesStream = Files.lines(Paths.get(getClass().getResource("/spamLinks.txt").toURI()))) {
```

Comment:
Just a suggestion because I think you didn't see what I had said; this is what Dynxsty suggested but I'm of the opinion that what I said would be the better approach.

Reply:
yeah